bika/health/utils.py

```diff
@@ -163,3 +167,53 @@ def handle_after_submit(context, request, state):
     else:
         status_id = "success"
     return status_id
+
+
+def get_age_ymd(birth_date, to_date=None):
+    """Returns the age at to_date if not None. Otherwise, current age
+    """
+    delta = get_relative_delta(birth_date, to_date)
+    return to_ymd(delta)
+
+
+def get_relative_delta(from_date, to_date=None):
+    """Returns the relative delta between two dates. If to_date is None,
+    compares the from_date with now
+    """
+    from_date = to_datetime(from_date)
+    if not from_date:
+        raise TypeError("Type not supported: from_date")
+
+    to_date = to_date or datetime.now()
+    to_date = to_datetime(to_date)
+    if not to_date:
+        raise TypeError("Type not supported: to_date")
+
+    return relativedelta(to_date, from_date)
+
+
+def to_datetime(date_value, default=None, tzinfo=None):
+    if isinstance(date_value, datetime):
+        return date_value
+
+    # Get the DateTime
+    date_value = api.to_date(date_value, default=None)
+    if not date_value:
+        if default is None:
+            return None
+        return to_datetime(default, tzinfo=tzinfo)
+
+    # Convert to datetime and strip
+    return DT2dt(date_value).replace(tzinfo=tzinfo)
+
+
+def to_ymd(delta):
```

Comment:
Shouldn't the date format depend on the region?

Reply:
This function is not about the date format. It receives a timedelta (difference between two dates) and returns a human-redable string like "13a 2m 1d". It is mainly used to display the age of a patient, not the date of birth.